src/__tests__/autocapture.js

```diff
@@ -42,6 +42,9 @@ describe('Autocapture system', () => {
             password.setAttribute('type', 'password')
             password.value = 'password val'
 
+            angularDiv = document.createElement('div')
```

Comment:
Given that this element is only used in one test, it's probably better to roll it into the specific test rather than expose it top-level. Makes the test more self-contained.